tactics/hints.ml

```diff
@@ -669,7 +677,20 @@ let try_head_pattern c =
 
 let with_uid c = { obj = c; uid = fresh_key () }
 
+let secvars_of_idset s =
+  Id.Set.fold (fun id p ->
+      if is_section_variable id then
+        Id.Pred.add id p
+      else p) s Id.Pred.empty
+
```

Comment:
No, you're right. I don't see how we could ever test that if we allow tampering.